libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteLine.kt

```diff
@@ -1505,6 +1527,67 @@ internal class MapRouteLine(
                 else -> y2
             }
         }
+
+        fun getRouteLineTrafficExpressionData(route: DirectionsRoute):
```

Comment:
Could you explain what the new methods do in a doc entry?

Reply:
done